Code/Tools/AssetProcessor/Platform/Linux/native/FileWatcher/FileWatcher_linux.cpp

```diff
@@ -15,155 +18,133 @@
 #include <sys/inotify.h>
 
 
-static constexpr int s_handleToFolderMapLockTimeout = 1000;      // 1 sec timeout for obtaining the handle to folder map lock
-static constexpr size_t s_iNotifyMaxEntries = 1024 * 16;         // Control the maximum number of entries (from inotify) that can be read at one time
-static constexpr size_t s_iNotifyEventSize = sizeof(struct inotify_event);
-static constexpr size_t s_iNotifyReadBufferSize = s_iNotifyMaxEntries * s_iNotifyEventSize;
+static constexpr size_t s_inotifyMaxEntries = 1024 * 16;         // Control the maximum number of entries (from inotify) that can be read at one time
```

Comment:
It seems like we should read this from a Linux specific setting located in `<engine-root>/Registry/Platform/Linux`?